### ISSUE TITLE
Build failed with Node > 5

### ISSUE THREAD

**Issue by Gillespie59** [Gillespie59/eslint-plugin-angular#395]
With these errors : 

```
/home/travis/build/Gillespie59/eslint-plugin-angular/rules/utils/utils.js
  0:0  error  Parsing error: Cannot read property 'ecmaFeatures' of undefined
/home/travis/build/Gillespie59/eslint-plugin-angular/test/utils/commonFalsePositives.js
  0:0  error  Parsing error: Cannot read property 'ecmaFeatures' of undefined
/home/travis/build/Gillespie59/eslint-plugin-angular/test/utils/testUtils.js
  0:0  error  Parsing error: Cannot read property 'ecmaFeatures' of undefined
/home/travis/build/Gillespie59/eslint-plugin-angular/test/utils/utils.js
  0:0  error  Parsing error: Cannot read property 'ecmaFeatures' of undefined
```

**Comment by Gillespie59** [comment]
Is now OK without doing anything
